sunpy/net/helioviewer.py

```diff
@@ -75,11 +110,22 @@ def get_closest_image(self, date, **kwargs):
         >>> print(metadata['date'])  # doctest: +REMOTE_DATA
         2012-01-01 00:00:07
         """
+
+        if('sourceId' in kwargs):
```

Comment:
We should make sure that sourceId has a lower case `i` for keywords and the documentation.